hathor/transaction/util.py

```diff
@@ -28,6 +28,10 @@ def int_to_bytes(number: int, size: int, signed: bool = False) -> bytes:
     return number.to_bytes(size, byteorder='big', signed=signed)
 
 
+def bytes_to_int(data: bytes, signed: bool = False) -> int:
```

Comment:
Missing docstring.

Reply:
Added in https://github.com/HathorNetwork/hathor-core/pull/555/commits/0a4cb8cabcfac6f176d3b1556dc6997393c6259c